README.md

```diff
@@ -158,6 +158,6 @@ If you like this project please show support by staring ⭐️.
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](/LICENSE.txt) licensed.
+This project is [MIT](https://github.com/Obimbo07/tdn/blob/feature/forms/LICENSE.txt) licensed.
```

Comment:
[REQUIRED] I am still being redirected to a 404 page kindly fix this. Ensure that your README and LICENSE files are in the same folder and link with something like this `./LICENSE.txt`. The problem lies in the route path.